src/shared/components/radar/radar.component.tsx

```diff
@@ -169,13 +171,15 @@ export const Radar = ({
 
   useEffect(() => {
     if (quadrantSectors && areaLabels) {
+      console.log('activeQuadrant', activeQuadrant);
```

Comment:
Console.log pewnie do usunięcia

Reply:
Tak to jest jak się nie ma eslinta przy commitowaniu 😄